joomla/joomla-cms
# [4.0] Fields/editor/none as custom element

Pull Request for Issue # .

### Summary of Changes
Textarea (AKA editor none) implementing the Joomla API for editors 


### Testing Instructions
Install from this PR's branch. Change the default editor to none. Check that the buttons work and they do:
- insert at the start of the text
- insert at the end of the text
- after making some selection and trying some of the buttons (the value from the xtd-button should replace the texture selection).

Mind that modals are not closing, it's a known issue! 


### Expected result



### Actual result



### Documentation Changes Required
